MOxUnit/@MOxUnitFunctionHandleTestCase/run.m

```diff
@@ -16,6 +16,7 @@
 %
 % NNO 2015
 
+    start_tic = tic;
```

Comment:
I was going to split the PR, but then the XML code became contingent on the timing being available. I can still split them if you prefer, but the XML code will then need another PR (or there will be XML code in the timing PR).

Using `tic` is fine because when you set a variable equal to a tic it goes into a different timer than the global last tic.

Demonstration:
```matlab
disp('start tic test');
tic;
pause(1);
toc;
inner_tic = tic;
pause(1.5);
toc(inner_tic);
toc;
disp('end tic test');
```
Output:
```
start tic test
Elapsed time is 1.004555 seconds.
Elapsed time is 1.507640 seconds.
Elapsed time is 2.522680 seconds.
end tic test
```

I would be happy with whatever gives the best result. I don't know if tic is more accurate than clock, but tic is certainly easier to use.